dotCMS/src/main/java/com/dotcms/rendering/velocity/services/TemplateLoader.java

```diff
@@ -0,0 +1,102 @@
+package com.dotcms.rendering.velocity.services;
+
+
+import com.dotcms.rendering.velocity.util.VelocityUtil;
+
+import com.dotmarketing.beans.Identifier;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.business.CacheLocator;
+import com.dotmarketing.business.DotStateException;
+import com.dotmarketing.business.VersionableAPI;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotSecurityException;
+import com.dotmarketing.portlets.templates.model.Template;
+import com.dotmarketing.util.Constants;
+import com.dotmarketing.util.Logger;
+import com.dotmarketing.util.PageMode;
+
+import java.io.InputStream;
+
+import org.apache.velocity.runtime.resource.ResourceManager;
+
+/**
+ * @author will
+ *
+ *         To change this generated comment edit the template variable "typecomment":
+ *         Window>Preferences>Java>Templates. To enable and disable the creation of type comments go
+ *         to Window>Preferences>Java>Code Generation.
+ */
+public class TemplateLoader implements DotLoader {
+
+
+
+    public InputStream buildVelocity(Template template, PageMode mode, String filePath)
+            throws DotStateException, DotDataException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'com.dotmarketing.business.DotStateException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)